index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Encode+Sans+Condensed&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <title>CV</title>
+</head>
+<body>
+<div class="wrapper">
+  <header class="header">
+    <div class="photo">
+      <img src="photo1.png">
+    </div>
+    <div class="personal-data">
+      <div class="name-wrapper">
+        <span class="name">John Doe</span>
+      </div>
+      <div class="contact-data">
+        <div class="contact-container">
+          <div class="contact">3009 Boulevard of Broken Dreams, Oakland, CA
+          </div>
+          <div class="phone">phone: +(0) 1 2345 555</div>
+          <div class="email">email: john@doe.com</div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main class="content">
+    <div class="info">
+      <div class="about">
+        ABOUT ME
+      </div>
+      <div class="about-text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ut tortor eu lectus consectetur mattis. Cras a turpis ut leo auctor auctor. Etiam nulla erat, gravida vel elit in, scelerisque laoreet erat. Phasellus rutrum velit dolor, quis porta tortor aliquam non. Nunc in risus faucibus, ultrices tortor sed, elementum augue.
+      </div>
+      <div class="about-additional">
+        Proin dignissim, eros ac consectetur venenatis, lorem nibh iaculis ligula, id suscipit orci est auctor nibh. Cras placerat non ligula eget ultrices. Integer sed ultrices nunc. Integer quis luctus massa.
+      </div>
+      <div class="skills">SKILLS</div>
+      <div class="skills-text">
+        <div class="skills-container">
+          <div class="graphic">Graphic Design</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="developer">Web Develop</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="lorem">Lorem Ipsum</div>

```

Comment:
You added a class for every possible element in the HTML, while you are not using all of them. Please get rid of the redundant classes.

Reply:
Reduced:)